homework_02/Martin_Georgiev_13/Templates/trains.html

```diff
@@ -17,43 +17,57 @@
         margin: 0 auto;
 
     }
-    </style>
+</style>
     
-    <body>
-        <div id="time">
-            <div style= "float: center;">
-    
-                <h1>Current time : {{ cur_date }}-{{ cur_month }}-{{ cur_year }} | {{ cur_time }}</h1>
-            </div>
+<body>
+    <br>
+    <div id="time">
+        <div style= "float: center;">
+
+            <h1>Current time : {{ day }}-{{ month }}-{{ year }} | {{ time }}</h1>
```

Comment:
The format here is good. You have maintained the identation, so, I have no objections here.